src/main/java/net/sf/jabref/logic/cleanup/RenamePdfCleanup.java

```diff
@@ -83,7 +89,12 @@ public RenamePdfCleanup(boolean onlyRelativePaths, BibDatabaseContext databaseCo
             }
 
             //do rename
-            boolean renameSuccessful = FileUtil.renameFile(expandedOldFilePath, newPath);
+            boolean renameSuccessful = false;
+            try {
+                renameSuccessful = FileUtil.renameFile(Paths.get(expandedOldFilePath), Paths.get(newPath));
```

Comment:
I think it would be better to place the try/catch construct in ``FileUtil.renameFile``, because the exception is actually thrown by ``Files.move(...)``